modules/configmap-create.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * builds/setting-up-trusted-ca
+
+[id="configmap-create-{context}"]
+= Creating a ConfigMap
+
+You can use the following command to create a ConfigMap from
+directories, specific files, or literal values.
+
+.Procedure
+
+* Create a ConfigMap:
+
+----
+$ oc create configmap <configmap_name> [options]
```

Comment:
The command should be like：
$oc create configmap registry-config --from-file=docker-registry-default.apps.0128-49c.qe.rhcloud.com=ca.crt -n **openshift-config**

Reply:
I think this part is duplicated with above part.